docker/jacctck.sh

```diff
@@ -71,6 +71,10 @@ sed -i 's#jacc\.host=.*#jacc.host=localhost#g' ts.jte
 sed -i "s#^report.dir=.*#report.dir=$TCK_HOME/${TCK_NAME}report/${TCK_NAME}#g" ts.jte
 sed -i "s#^work.dir=.*#work.dir=$TCK_HOME/${TCK_NAME}work/${TCK_NAME}#g" ts.jte
 
+if [[ "$PROFILE" == "web" || "$PROFILE" == "WEB" ]]; then
```

Comment:
I think ts.jte in install/bin/jacc already has javaee.level=web! I think it was supposed to be full.

Reply:
thanks for reviewing, I have removed changes for docker/jacctck.sh